### PR TITLE
Fix/liwo 709 combined click on map

### PR DESCRIPTION
**Pull request by vvandijck**

### CODE REVIEW

File: src/lib/leaflet-utils/popup.js [hunk 1]
```diff
@@ -26,3 +28,34 @@ export function showLayerInfoPopup ({ map, layerId, unit, selectedLayer, positio
...
+    })
+  }
+
+  fetch(url, options)
+    .then(response => response.json())
+    .then(({ value }) => {
+      if (value !== null) {
+        L.popup()
+          .setLatLng(coordinates)
+          .setContent(`${value}`)
```
Comment (vvandijck) [inline src/lib/leaflet-utils/popup.js:56]: We could also do `const x = value.toString()` because the value is a Number.

Comment (sjoerdbeentjes) [reply src/lib/leaflet-utils/popup.js:56]: yeah let's go for that option (`toString`)

File: public/config/webconfig-old.json [hunk 2]
```diff
@@ -18,6 +18,7 @@
   "MANUAL_URL": "https://www.helpdeskwater.nl/onderwerpen/applicaties-modellen/applicaties-per/watermanagement/watermanagement/liwo",
   "_comment5": "// URL for combining scenario's",
   "HYDRO_ENGINE_URL": "https://green-dot-hydro-engine.ey.r.appspot.com/v2",
+  "HYDRO_ENGINE_URL_V2": "https://green-dot-hydro-engine.appspot.com",
```
Comment (sjoerdbeentjes) [inline public/config/webconfig-old.json:21]: This is a bit confusing 😅 can we swap this?

**Review by sjoerdbeentjes — CHANGES_REQUESTED** [review]
Looking good!

added 1 small comment, let me know what you think of it

File: src/lib/leaflet-utils/popup.js [hunk 3]
```diff
@@ -26,3 +28,34 @@ export function showLayerInfoPopup ({ map, layerId, unit, selectedLayer, positio
...
+  const options = {
+    method: 'POST',
+    headers: { 'Content-Type': 'application/json' },
+    body: JSON.stringify({
+      imageId: 'projects/dgds-gee/glossis/waterlevel/glossis_waterlevel_20200301000000',
+      bbox: {
+        type: 'Point',
+        coordinates: [lat, lng]
+      },
+      band: 'water_level_surge'
```
Comment (sjoerdbeentjes) [inline src/lib/leaflet-utils/popup.js:46]: should this be hardcoded?

**Pull request merged**